pkgs/os-specific/bsd/freebsd/pkgs/mkDerivation.nix

```
@@ -77,6 +86,13 @@ lib.makeOverridable (
 
         # don't set filesystem flags that require root
         NO_FSCHG = "yes";
+
+        HAVE_LLVM = lib.optionalString stdenv'.cc.isClang or false (
```

Probably this? Looks *really* weird otherwise.

```suggestion
        HAVE_LLVM = lib.optionalString (stdenv'.cc.isClang or false) (
```

(Same below)

---

I don't really like adding extra parens — I'd rather somebody be surprised and learn something about how Nix works that will help them in future, especially since there's not really any other plausible reading in this case.

---

Fine with me